test/paths.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"bufio"
+	"os"
+	"strings"
+)
+
+// GetRelativeDependencyPath returns the relative path for the specified module name
+//
+// moduleName string - module name or substring
+//
+// return string
+func GetRelativeDependencyPath(moduleName string) string {
+	var crdPath string
+	readFile, err := os.Open(GetGoModFilepath())
+
+	if err != nil {
+		panic(err)
+	}
+	defer readFile.Close()
+
+	scanner := bufio.NewScanner(readFile)
+	for scanner.Scan() {
+		if strings.Contains(scanner.Text(), moduleName) {
+			crdPath = strings.Trim(string(scanner.Text()), "\t")
+			break
+		}
+	}
+
+	return strings.ReplaceAll(crdPath, " ", "@")
+}
+
+// GetGoModFilepath returns the go.mod file location for the current project or empty string when file is not found
+//
+// return string
+func GetGoModFilepath() string {
+	var parentDir string
+	currentWorkDir, _ := os.Getwd()
+
+	for _, dir := range []string{"..", ".."} {
```

Comment:
We can keep the function (check my other comment about relativity of the path), but I don't like that it's limited to two levels. I would use an infinite loop and walk backwards using filepath.Dir until you find the file or until the function returns `.` (I think this is the return when you are in the root, but double check).